src/test/java/reposense/model/RepoConfigurationTest.java

```diff
@@ -185,6 +188,48 @@ public void repoConfig_ignoresStandaloneConfigInCli_success()
         TestUtil.compareRepoConfig(expectedConfig, actualConfig);
     }
 
+    @Test
+    public void repoConfig_ignoresStandaloneConfigInCliOverridesCsv_success()
+            throws HelpScreenException, IOException, ParseException {
+
+        RepoConfiguration repoDeltaExpectedConfig = new RepoConfiguration(
+                new RepoLocation(TEST_REPO_DELTA), "master");
+        repoDeltaExpectedConfig.setFormats(FileType.convertFormatStringsToFileTypes(CLI_FORMATS));
+        repoDeltaExpectedConfig.setStandaloneConfigIgnored(true);
+
+        RepoConfiguration repoCharlieExpectedConfigWithoutIgnoreStandalone = new RepoConfiguration(
+                new RepoLocation(TEST_REPO_CHARLIE));
+        RepoConfiguration repoCharlieExpectedConfigWithIgnoreStandalone = new RepoConfiguration(
+                new RepoLocation(TEST_REPO_CHARLIE));
+        repoCharlieExpectedConfigWithIgnoreStandalone.setStandaloneConfigIgnored(true);
+
+        String inputWithoutIgnoreStandaloneFlag = new InputBuilder().addConfig(IGNORE_STANDALONE_FLAG_OVERRIDE_CSV_TEST)
+            .build();
+        String inputWithIgnoreStandaloneFlag = new InputBuilder().addConfig(IGNORE_STANDALONE_FLAG_OVERRIDE_CSV_TEST)
+            .addIgnoreStandaloneConfig()
+            .build();
+
+        CliArguments argsWithoutIgnoreStandaloneFlag = ArgsParser.parse(
+            translateCommandline(inputWithoutIgnoreStandaloneFlag));
+        CliArguments argsWithIgnoreStandaloneFlag = ArgsParser.parse(
+                translateCommandline(inputWithIgnoreStandaloneFlag));
+
+        List<RepoConfiguration> repoConfigsWithSomeIgnoringStandaloneConfig =
+                RepoSense.getRepoConfigurations((ConfigCliArguments) argsWithoutIgnoreStandaloneFlag);
+        List<RepoConfiguration> repoConfigsWithAllIgnoringStandaloneConfig =
+                RepoSense.getRepoConfigurations((ConfigCliArguments) argsWithIgnoreStandaloneFlag);
+
+        RepoConfiguration repoCharlieActualConfig = repoConfigsWithSomeIgnoringStandaloneConfig.get(0);
+        RepoConfiguration repoDeltaActualConfig = repoConfigsWithSomeIgnoringStandaloneConfig.get(1);
+        TestUtil.compareRepoConfig(repoCharlieActualConfig, repoCharlieExpectedConfigWithoutIgnoreStandalone);
+        TestUtil.compareRepoConfig(repoDeltaActualConfig, repoDeltaExpectedConfig);
+
+        repoCharlieActualConfig = repoConfigsWithAllIgnoringStandaloneConfig.get(0);
+        repoDeltaActualConfig = repoConfigsWithAllIgnoringStandaloneConfig.get(1);
+        TestUtil.compareRepoConfig(repoCharlieActualConfig, repoCharlieExpectedConfigWithIgnoreStandalone);
+        TestUtil.compareRepoConfig(repoDeltaActualConfig, repoDeltaExpectedConfig);
```

Comment:
For comparison tests, expected should come before actual as per the current convention. So let's standardize it

```java
TestUtil.compareRepoConfig(EXPECTED, ACTUAL);
```

Reply:
Done